pkg/translator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -55,7 +55,7 @@ func FromMetrics(md pmetric.Metrics, settings Settings) (tsMap map[string]*promp
 				mostRecentTimestamp = maxTimestamp(mostRecentTimestamp, mostRecentTimestampInMetric(metric))
 
 				if !isValidAggregationTemporality(metric) {
-					errs = multierr.Append(errs, errors.New("invalid temporality and type combination"))
+					errs = multierr.Append(errs, fmt.Errorf("invalid temporality and type combination. %s is dropped", metric.Name()))
```

Comment:
The error message already contains information that metrics will be dropped (`Exporting failed. The error is not retryable. Dropping data. ...`). Let's maybe keep the name of the dropped metric only, e.g. `fmt.Errorf("invalid temporality and type combination for %q  metric", metric.Name()))`

Reply:
Updated.